hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -492,17 +492,14 @@
     {
       name: chip_sw_data_integrity
       desc: '''
-            Verify the alert signaling mechanism due to data integrity violation.
-
-            An automated SW test which performs the following for each IP:
-            - Pick a CSR to write.
-            - Corrupt a random control / data / integrity bit at the CPU source using SV force.
-            - Verify that the device detects the integrity violation causing an alert.
-            - Verify the alert upto the NMI stage.
-            - Now pick a CSR to read.
-            - Corrupt a random control / data / integrity bit at the device using SV force.
-            - Verify that the CPU detects the integrity violation causing an alert.
-            - Verify the alert upto the NMI stage.
+            Verify the alert signaling mechanism due to memory data integrity violation.
+
+            An SW test which performs the following on ret_sram to verify the memory end to end
+            integrity scheme.
+            - Corrupt a random data / integrity bit in the memory using SV force.
+            - SW reads that address and the corrupted data is sent to ibex.
+            - Verify that ibex detects the integrity violation and triggers an alert.
+            - Verify the alert upto the NMI stage and alert cause is from ibex.
```

Comment:
In the other escalation reset tests we go all the way to a full reset, yet we check that an NMI is generated. What I mean is that "upto the NMI stage" is optional, and "upto the NMI stage or full reset" may end up being more accurate.

Reply:
Update. Thanks Matute.

Reply:
Thanks